packages/reporters/application-profiler/src/ApplicationProfilerReporter.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+import invariant from 'assert';
+import nullthrows from 'nullthrows';
+import {Reporter} from '@parcel/plugin';
+import {Tracer} from 'chrome-trace-event';
+
+// We need to maintain some state here to ensure we write to the same output, there should only be one
```

Comment:
Sidenote: This is exactly why `options.instanceId` was added. But in reality, this probably never happens anyway